stable/traefik/values.yaml

```diff
@@ -26,3 +26,6 @@ dashboard:
   domain: traefik.example.com
 gzip:
   enabled: true
+kubernetes:
+  namespaces: ~
+  labelSelector: ~
```

Comment:
What's the meaning and effective result of the `~` here? I inferred it means "all," but I can't find in the Traefik docs where it says that. I think that rather than set these to ~, it might be better to leave these commented (and therefore undefined) by default. It will be more consistent with other pending PRs.